src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+    boolean smallBlindCallPreFlop = true;
+    public EquityEvaluator equityEvaluator;
+    public ResultHandOut resultHandOut;
+    public GameResult gameResult;
+    boolean firstHand = true;
+
+    public GamePlay() {
+        this.deck = new Deck();
+        this.boardAndCards = new MakeBoardAndCards(deck);
+        this.result = boardAndCards.makeHandsBoard();
+        this.player1 = new Player(100, 0, result.getHand1(), "player1");
+        this.player2 = new Player(100, 0, result.getHand2(), "player2");
+        this.gameState = new GameState(GamePhase.PREFLOP, 0);
+        playerList = new ArrayList<>();
+        playerList.add(player1);
+        playerList.add(player2);
+        this.equityEvaluator = new EquityEvaluator(deck, result.getHand1(), result.getHand2(), result.getBoard());
+        this.gameResult = new GameResult();
+    }
+
+    // wykryc kto wygra na river (bez all inow) udpate staka i rozpoczęcie nowego handa, potem uprzątnać to co się da
+    // commit i push wysłać linka zadanie3 do git huba
+    public void gameProgress() {
+        int i = 0;
+        while (playerList.size() > 1) {
+            if (firstHand) {
+            } else {
+                try {
+                    Thread.sleep(4000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+            for (int n = 0; n < 20; n++) {
+                System.out.println();
+            }
+            System.out.println(player1.getStackPlayer());
+            System.out.println(player2.getStackPlayer());
+            if (i % 2 == 0) {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player1);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player2);
+                System.out.println("Zaczyna " + player2.getNamePlayer());
+                preFlopSmallBlind(player1, player2);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+            } else {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player2);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player1);
+                System.out.println("Zaczyna " + player1.getNamePlayer());
+                preFlopSmallBlind(player2, player1);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+            }
+            firstHand = false;
+            i++;
```

Comment:
to od wywalenia i w miejsce tego dajesz 
Player temp = bigBlindPlayer
bigBlindPlayer = smallBlindPlayer
smallBlindPlayer = temp